### PR TITLE
Fixes #58 Move to CVODES 4.0

### LINKED ISSUE #58: Move to CVODES 4.0
Move to CVODES 4.0

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ before_build:
   - nuget sources add -name bddhelper -source https://ci.appveyor.com/nuget/ospsuite-bddhelper 
   - nuget sources add -name utility -source https://ci.appveyor.com/nuget/ospsuite-utility 
   - nuget sources add -name funcparser -source https://ci.appveyor.com/nuget/ospsuite-funcparser 
-  - nuget sources add -name cvodes -source https://ci.appveyor.com/nuget/ospsuite-simmodel-solver-cvodes-282 
+  - nuget sources add -name cvodes -source https://ci.appveyor.com/nuget/ospsuite-simmodel-solver-cvodes
```
Comment (Yuri05) [inline appveyor.yml]: @msevestre Is that correct address of the nuget feed for the new solver?

File: tests/OSPSuite.SimModel.Tests/src/SimulationSpecs.cpp [hunk 2]
```diff
@@ -267,6 +267,7 @@ namespace SimulationTests
 
     public:
         [TestAttribute]
+        [Ignore("")]
```
Comment (Yuri05) [inline tests/OSPSuite.SimModel.Tests/src/SimulationSpecs.cpp]: deactivated all tests with band linear solver. Is not used by the OSP anyway and will be probably removed soon.
Instead, I'll try to integrate new Sparse linear solver availavle in CVODES 4.

File: src/OSPSuite.SimModel/OSPSuite.SimModel.vcxproj [hunk 3]
```diff
@@ -267,7 +267,10 @@ copy "$(SolverPath)bin\native\$(Platform)\$(Configuration)\*.dll" "$(OutputPath)
       <CompileAsManaged Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">false</CompileAsManaged>
       <CompileAsManaged Condition="'$(Configuration)|$(Platform)'=='Release|x64'">false</CompileAsManaged>
     </ClCompile>
-    <ClCompile Include="src\CppODEExporter.cpp" />
+    <ClCompile Include="src\CppODEExporter.cpp">
+      <CompileAsManaged Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">false</CompileAsManaged>
+      <CompileAsManaged Condition="'$(Configuration)|$(Platform)'=='Release|x64'">false</CompileAsManaged>
+    </ClCompile>
     <ClCompile Include="Src\DESolver.cpp">
       <CompileAsManaged Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">false</CompileAsManaged>
```
Comment (msevestre) [inline src/OSPSuite.SimModel/OSPSuite.SimModel.vcxproj]: Why is this still here? Should'nt we remove all win32 stuff?

File: tests/OSPSuite.SimModel.Tests/src/SimulationSpecs.cpp [hunk 4]
```diff
@@ -611,6 +614,7 @@ namespace SimulationTests
...
```
Comment (msevestre) [inline tests/OSPSuite.SimModel.Tests/src/SimulationSpecs.cpp]: That's useful message :)

Comment (Yuri05) [reply tests/OSPSuite.SimModel.Tests/src/SimulationSpecs.cpp]: lol. with 99% probabilty this functionality will be removed soon, so the tests will be removed as well

**Pull request merged**